### DISCUSSION TITLE
Cannot create issues or pull requests

### DISCUSSION THREAD

**Discussion by amodelbello** [magit/forge#581] [Support] [answered]
Magit 20230603.1122 [>= 3.3.0.50-git], Transient 0.4.1, Git 2.34.1, Emacs 28.2, gnu/linux
forge-20230522.1054

I'm unable to create issues or pull requests. Every attempt results in the error:
```error in process filter: Failed to submit post: (error http 404 ((message . "Not Found") (documentation_url . "https://docs.github.com/rest/reference/pulls#create-a-pull-request")))```

It's for a public github repo. I'm the owner of the repository and I'm pretty sure it's not an issue with my token. I even went so far as to create a token with ALL permissions enabled and am still experiencing the issue. I've tried both classic and fine-grained tokens.

The fetch and list commands seem to work fine.

**Comment by tarsius** [comment, marked as answer]
I am guessing you renamed the repository or account. While the web interface redirects to the new destination, the API does not. (You mention just one repository. Have you tried with another?)

**Reply by amodelbello** [reply]
That absolutely was my issue! Once I updated my remote url to the new name it worked. Thanks so much for the immediate solution.
